docs/pages/machine-id/manifesto.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Machine ID Manifesto
+description: A manifesto for Machine Identity
+---
+
+As time goes on, machines and services are completing increasingly privileged
+work, crossing the complex boundaries between networks and organizations. AI
```

Comment:
> crossing the complex boundaries between networks and organizations.

I'm not sure if I understand this. What would be an example?

Reply:
Ah, gotcha. Should we turn this first sentence into:

> Machines and services are not only completing increasingly privileged
work, they need to communicate more with machines and services on other networks, including networks that belong to different organizations. This introduces a new class of problems around federating trust across organization and network boundaries.

Reply:
Hmm - I agree I could be clearer but I think I need to try some different phrasing. Something about the "not only" makes it feel to me as if the first clause is less significant than the second clause, when I'd probably prefer to emphasise the first clause (machines and services are completing increasingly privileged). I'll have a think.

Reply:
Rewritten.